src/components/accordion/README.md

```diff
@@ -19,23 +19,25 @@ import { UiAccordion } from '@dreipol/vue-ui/src/components';
 
 ## Example
 ```vue
-<accordion>
+<!-- basic usage -->
+<ui-accordion>
```

Comment:
I am not sure but in other places we have also unprefixed examples. Could you please check them and update the doc accordingly everywhere. thanks

Reply:
We had both versions. I've streamlined it to always use `<ui-xy...`